Documentation Standard Updates (#43)
fivetran/dbt_mailchimp: README.md

# Mailchimp Transformation dbt Package ([Docs](https://fivetran.github.io/dbt_mailchimp/))

## What does this dbt package do?
- Produces modeled tables that leverage Mailchimp data from [Fivetran's connector](https://fivetran.com/docs/applications/mailchimp) in the format described by [this ERD](https://fivetran.com/docs/applications/mailchimp#schemainformation) and builds off the output of our [Mailchimp source package](https://github.com/fivetran/dbt_mailchimp_source).
- Transforms the 'recipient' and 'activity' tables into analytics-ready models and use that data to provide aggregate metrics about campaigns, automations, lists, members, and segments.

<!--section="mailchimp_transformation_model-->
- Generates a comprehensive data dictionary of your source and modeled Mailchimp data through the [dbt docs site](https://fivetran.github.io/dbt_mailchimp/#!/overview).

> The following table provides a detailed list of all models materialized within this package by default.
> > TIP: See more details about these models in the package's [dbt docs site](https://fivetran.github.io/dbt_mailchimp/#!/overview?g_v=1).
> 
> | **Model**                | **Description**                                                                                                                                |

Author: ```suggestion
The following table provides a detailed list of all tables materialized within this package by default.
> TIP: See more details about these tables in the package's [dbt docs site](https://fivetran.github.io/dbt_mailchimp/#!/overview?g_v=1).

| **Table**                | **Description**                                                                                                                                |
```

```yml
models:

> mailchimp:

Author: ```suggestion
  mailchimp:
```

    +schema: my_new_schema_name # leave blank for just the target_schema

> mailchimp_source:

Author: ```suggestion
  mailchimp_source:
```
